pandas_gbq/gbq.py

```diff
@@ -847,44 +738,31 @@ def _parse_data(schema, rows):
     fields = schema['fields']
     col_types = [field['type'] for field in fields]
     col_names = [str(field['name']) for field in fields]
-    col_dtypes = [dtype_map.get(field['type'], object) for field in fields]
+    col_dtypes = [
+        dtype_map.get(field['type'].upper(), object)
+        for field in fields
+    ]
+    print(fields)
```

Comment:
Do we want to print all the fields here?

If we do: elsewhere, this uses `self._print`, which can can transition to logging at some point.

Reply:
Oops. Good catch. I added that to debug the "ignore mode when comparing schemas" logic. Removed.